website/registrations/utils.py

```diff
@@ -10,29 +10,34 @@
 from rest_framework.exceptions import NotFound, ValidationError
 
 from osf.models.licenses import NodeLicense
-from osf.models import AbstractNode, RegistrationProvider, Subject, RegistrationSchema, Institution
+from osf.models import AbstractNode, RegistrationProvider, RegistrationSchema, Institution
 from website import settings
 
 
-METADATA_FIELDS = {'title': {'format': 'string', 'required': True},
-                   'description': {'format': 'string', 'required': True},
-                   'admin contributors': {'format': 'list', 'required': True},
-                   'read-write contributors': {'format': 'list'},
-                   'read-only contributors': {'format': 'list'},
-                   'bibliographic contributors': {'format': 'list'},
-                   'category': {'format': 'string'},
-                   'affiliated institutions': {'format': 'list'},
-                   'license': {'format': 'object', 'required': True},
-                   'subjects': {'format': 'list', 'required': True},
-                   'tags': {'format': 'list'},
-                   'project guid': {'format': 'string'},
-                   'external id': {'format': 'string'}}
-CONTRIBUTOR_METADATA_FIELDS = ['admin contributors',
-                               'read-write contributors',
-                               'read-only contributors',
-                               'bibliographic contributors']
+METADATA_FIELDS = {'Title': {'format': 'string', 'required': True},
+                   'Description': {'format': 'string', 'required': True},
+                   'Admin Contributors': {'format': 'list', 'required': True},
+                   'Read-Write Contributors': {'format': 'list'},
+                   'Read-Only Contributors': {'format': 'list'},
+                   'Bibliographic Contributors': {'format': 'list'},
+                   'Category': {'format': 'string'},
+                   'Affiliated Institutions': {'format': 'list'},
+                   'License': {'format': 'object', 'required': True},
+                   'Subjects': {'format': 'list', 'required': True},
+                   'Tags': {'format': 'list'},
+                   'Project GUID': {'format': 'string'},
+                   'External ID': {'format': 'string'}}
+CONTRIBUTOR_METADATA_FIELDS = ['Admin Contributors',
+                               'Read-Write Contributors',
+                               'Read-Only Contributors',
+                               'Bibliographic Contributors']
+
+CATEGORY_REVERSE_LOOKUP = {display_name: name for name, display_name in settings.NODE_CATEGORY_MAP.items()}
+
 MAX_EXCEL_COLUMN_NUMBER = 16384
 
+STORE = None
```

Comment:
As we've discussed, I have concern about `global` `STORE` which I need to test. Just a comment for future reference when we come back.